docs/guides/Guide-Basic-Example.md

```diff
@@ -0,0 +1,235 @@
+# Hello Mobile Navigation
+
+Let's use React Navigation to build a simple chat application for Android and iOS.
+
+## Setup and Installation
+
+First, make sure you're [all set up to use React Native](http://facebook.github.io/react-native/docs/getting-started.html). Next, create a new project and add `react-navigation`:
+
+```sh
+# Create a new React Native App
+react-native init SimpleApp
+cd SimpleApp
+
+# Install the latest version of react-navigation from npm
+npm install --save react-navigation
+
+# Run the new app
+react-native run-android
+# or:
+react-native run-ios
+```
+
+If you are using `create-react-native-app` instead of `react-native init`, then:
+
+```sh
+# Create a new React Native App
+create-react-native-app SimpleApp
+cd SimpleApp
+
+# Install the latest version of react-navigation from npm
+npm install --save react-navigation
+
+# Run the new app
+npm start
+
+# This will start a development server for you and print a QR code in your terminal.
+```
+
+Verify that you can successfully see the bare sample app run on iOS and/or Android:
+
+```phone-example
+bare-project
+```
+
+We want to share code on iOS and Android, so let's delete the contents of `index.ios.js` and `index.android.js` and replace it with `import './App';` - after which, we need to create the new file for our app implementation, `App.js` (if you used `create-react-native-app` this has been already done)
```

Comment:
`index.ios.js` and `index.android.js` were deprecated with the latest React Native release https://github.com/facebook/react-native/releases/tag/v0.49.0

Reply:
That said maybe rephrasing saying something like `(in older versions, you had two separate files, index balb alb alba and bla bla bla)`